scripts/jira/cloner.py

```diff
@@ -75,7 +87,7 @@ def set_target_version(issue, version):
 def set_needs_fix_version_label(issue):
     labels = issue.fields.labels + ['needs-fix-version']
     issue.update(fields={
-        'labels': [{'name': x} for x in labels]
+        'labels': [x for x in labels]
```

Comment:
```suggestion
        'labels': labels,
```

Do we need a new list there?